docs/sphinx/user_guide/feature/policies.rst

```diff
@@ -147,28 +174,72 @@ CUDA Policies
 * ``cuda_block_y_loop`` - Maps loop iterations to cuda thread blocks in y dimension.
 * ``cuda_block_z_loop`` - Maps loop iterations to cuda thread blocks in z dimension.
 
-----------------------
-RAJA::region Policies
-----------------------
+.. _loop_elements-kernelpol-label:
 
-The following policies may only be used with the ``RAJA::region`` method.
+--------------------------------
+RAJA Kernel Execution Policies
+--------------------------------
 
-* ``seq_region_exec`` - Creates a sequential region.
-* ``omp_parallel_region_exec`` - Create an OpenMP parallel region.
+RAJA kernel policies are constructed with a combination of *Statements* and
+*Statement Lists* that forms a simple domain specific language that
+relies **solely on standard C++11 template support**. A RAJA Statement is an
+action, such as executing a loop, invoking a lambda, setting a thread barrier,
+etc. A StatementList is an ordered list of Statements that are composed
+to construct a kernel in the order that they appear in the kernel policy.
+A Statement may contain an enclosed StatmentList. Thus, a
+``RAJA::KernelPolicy`` type is simply a StatementList.
 
--------------------------
-RAJA::scan Policies
--------------------------
+The main Statements types provided by RAJA are ``RAJA::statement::For`` and
```

Comment:
Should be either: "main _statement_ types" or "main _Statement_ types".

Reply:
Done.